Annotations/HideFreeTextBorder/README.md

```diff
@@ -0,0 +1,59 @@
+# How to hide the border of free text annotation in MAUI PDF Viewer?
+
+This folder contains a sample project that demonstrates How to hide the border of free text annotation in MAUI PDF Viewer
+
+## Prerequisites
+1. .NET MAUI installed.
+2. Syncfusion.Maui.PdfViewer NuGet package.
+
+## Steps to achieve borderless free text annotations:
+
+### 1. Install Required NuGet Package
+
+Create a .NET MAUI application and install the `Syncfusion.Maui.PdfViewer` NuGet package using the NuGet Package Manager or the CLI.
+
+### 2. Initialize PDF Viewer in XAML
+
+1. **Add Namespace:**
+   Include the Syncfusion namespace in your `MainPage.xaml` file to use the PDF Viewer control.
+
+   ```xml
+   xmlns:syncfusion="clr-namespace:Syncfusion.Maui.PdfViewer;assembly=Syncfusion.Maui.PdfViewer"
+   ```
+
+2. **Add PDF Viewer Control:**
+   Insert the `SfPdfViewer` control into your layout and bind it to a document source for displaying PDF files.
+
+   ```xaml
+   <syncfusion:SfPdfViewer x:Name="pdfViewer" DocumentSource="{Binding PdfDocumentStream}"/>
+   ```
+
+### 3. Load a PDF Document
+
+Create a `PdfViewerViewModel` class and assign an embedded PDF file stream to the `PdfDocumentStream` property:
+
+```csharp
+public class PdfViewerViewModel
+{
+    public Stream PdfDocumentStream { get; private set; }
+
+    public PdfViewerViewModel()
+    {
+        // Set the embedded PDF stream
+        PdfDocumentStream = this.GetType().Assembly.GetManifestResourceStream("HideFreeTextBorder.Assets.PDF_Succinctly1.pdf");
+    }
+}
+```
+
+### 4. Customize Free Text Annotation
+
```

Comment:
For single line of code we may need to cover in UG instead of a KB or custom sample.

Custom sample or KB needs to be done for use case based approaches. Please convert to UG